### DISCUSSION TITLE
リンク先不備

### DISCUSSION THREAD

**Discussion by Brutus03** [increments/qiita-discussions#505] [要望・アイデア・不具合]
[公式イベント](https://qiita.com/official-events)のページから、Twitterキャンペーンのイベントを参照すると「数式を描画するライブラリのバージョンアップを行います」が表示されます。

リンク先は正しいですか？

**Comment by tanaka-asami415** [comment]
@Brutus03 
ご報告ありがとうございます🙏

誤った情報が公開されていたため、一旦非表示対応いたしました！
教えてくださりありがとうございます！🙇
